src/v/cluster/members_backend.cc

```diff
@@ -1010,6 +1010,9 @@ ss::future<> members_backend::reallocate_replica_set(
               meta.current_replica_set,
               meta.new_replica_set,
               error.message());
+            if (error == errc::update_in_progress) {
+                meta.state = reallocation_state::finished;
+            }
```

Comment:
Please add a code comment describing why we want to finish the relocation when there is a configuration update in progress alread.